nixos/tests/ceph-multi-node.nix

```diff
@@ -22,6 +22,8 @@ let
   };
   generateCephConfig = { daemonConfig }: {
     enable = true;
+    # 3 is the default, and we have only 2, so we get HEALTH_WARN
+    extraConfig."osd pool default size" = "2";
```

Comment:
We might just want to increase the number of OSDs, by providing another VM for it.

Reply:
I considered it. It seems a bit wasteful, but on the other hand, no catastrophe. I can do that :)

Reply:
Fixed.